src/components/Cart/CartListItem.jsx

```diff
@@ -95,16 +104,58 @@ const CartListItem = ({ cartItem }) => {
 					},
 				});
 			} else {
-				showToast(
-					"Failed to delete product. Please try again later.",
-					"error"
-				);
+				if (showToastAfterRemovingItem)
+					showToast(
+						"Failed to remove item from cart. Please try again later.",
+						"error"
+					);
 			}
 		} catch (error) {
 			console.log("Something went really wrong!");
 		}
 	};
 
+	const isItemInWishList = wishListItems.find(
+		(wishListItem) => wishListItem._id === _id
+	);
+
+	const handleMoveItemToWishList = async () => {
+		if (isItemInWishList) {
+			showToast("Item added to wishlist!", "success");
+			handleRemoveFromCart(false);
+		} else {
+			setIsOngoingNetworkCall(true);
```

Comment:
Appreciation: Great naming convention. Codes are readable and clean. Great work.

Reply:
Thank you!